app/components/windows/EditStreamInfo.vue

```diff
@@ -25,6 +25,41 @@
         :loading="searchingGames"
         @search-change="debouncedGameSearch"
         @input="onGameInput"/>
+        <div  v-if="areAvailableProfiles">
+        <div class="input-container">
+         <div class="input-label">
+         </div>
+         <div class="input-wrapper">
+           <div class="checkbox">
+             <input
+               type="checkbox"
+               v-model="useOptimizedProfile"
+             />
+             <label><span>Use optimized encoder settings for this game</span></label>
+           </div>
+         </div>
+       </div>
+       <div class="input-container select">
+         <div class="input-label">
+           <label>Profile</label>
+         </div>
+         <div class="input-wrapper">
+           <multiselect
+             :allowEmpty="false"
+             :options="profiles"
+             track-by="value"
+             :close-on-select="true"
+             label="description"
+             v-model="encoderProfile">
+             <template slot="option" scope="props">
+               <span>
+                 {{ props.option.description }}
+               </span>
+             </template>
+           </multiselect>
+         </div>
+       </div>
+       </div>
```

Comment:
Something weird is going on with this indentation.

Reply:
Yeal lol something went really wrong with the indentation. Should be cleaned up now.